app-admin/prelude-manager/Manifest

```diff
@@ -0,0 +1,8 @@
+AUX prelude-manager-3.0.0-conf.patch 595 SHA256 68bfdb85b3d9ce6909ed93bcc5f1fab4cf20af0dea7f47cdf652dbcd0efed950 SHA512 bca3ab760ab3e425c5e3b0a44865265ef508faf7b4d01ef7bf4749ccd93a950bea16e56a7fe93b0f4a09c548ca95ece3ddccf513186097557f6027e33b9fed68 WHIRLPOOL a92c1909a818db2f527b876b8a74d954c539bf5f28ac3bda1157f42a78ffc60356bbd20f5881cfffbbcc9c68b1fb82a6092f700e5727c46a7e738ffda452e05d
+AUX prelude-manager-3.0.0-configure.patch 2795 SHA256 96a767972c056361716501226ff541e0f5e878079ec81c5ddabd43ddcd2c3c54 SHA512 3d427e269450e3052174c2d1e1080f433396787e36d191aa52b3b785e702133a4ca42c96cbcfb29abb7a4793970ba789ee5aea67451b64eaa5d46f5787b22dda WHIRLPOOL 27c9ef9a9048da89543edc795f276d3de1fcc6ea388c67cce337f52dbcd1bb4b1743a0b03b40bf67dc1824319eb36870115e0a5dba5ea68a46161677fbfd1dd4
+AUX prelude-manager-3.0.0-run.patch 668 SHA256 578b30b75a8ea038aacde7c31d92721f0a61ad7405715405fbc90ededd92d08e SHA512 f01b5753fab4297050383dffcb57564290404e9ebb89b439475b1c2cdb015123a7ba5a106ce9dd0a6956db77619f40ed83145558a6f4648207e8d1f2eee8df63 WHIRLPOOL 3c739f79901da7c2b6ad9497cdfd039636bc064dfa4a1eed881a27b5a5d8c1016201e0050f9869e257e61ff0323936dca3cb85a3d528dc89c4e81b30cda73fc9
+AUX prelude-manager.run 133 SHA256 10fda7dada1c209ad0aadfab673e1bcc0bf2847e9e78dc0df110272b73866600 SHA512 f129cc45efb710d336dd177ba465f542aaac767579dafbbacbdc88b56fcf3b95eb219a9dff7863ca47475748b93d001218ff28a7e49b3060c62278a72666986e WHIRLPOOL 6371a588deaeed160c05f99579ade6b6c092f089f188a65a8e0317191dc7cde09ade8e8b83fd4fa2c346cb8d3422a514abdce0057327135b0ca5e9e41eecf412
+AUX prelude-manager.service 398 SHA256 1d11138ee19164a98958b270ca24456d330f1cbdae2b9afe129e1d85512763cd SHA512 bae09bbba400353a94627fa212258fca5bb17ee9de12817737c68b5dbb2c9441fe7f323f28442331fc45d6409e30e1a0c5f8b0077dd4f7bd63de8d86b934997a WHIRLPOOL eeb7d10f2394b75981014b8602b330731b32af536116a3e6eba3d2fc04f0bb329a6b3c8f2e1930eae20549e106f02eaee7d58835b3b342c5aac0ef9ec9cbe115
+DIST prelude-manager-3.0.0.tar.gz 1286459 SHA256 379de4c79ccdba4093e9c5a2b9e5c6cee8e8d8694e1358c3d248250c4957dc0b SHA512 4c16a353495179265375686ce9592c8cd044b4255060b78286466bd25f9b38634233c95af058e1c0b81e1b574ec9ada5b2db92d6d1f5594dfcddce572fd0fd67 WHIRLPOOL 35814a88e62e4bfdbd3e8c88f733cd304f8fb355d820ae9887da0c9e6b02936b7aed8d73a05cc458fbf01f7a9e48174d446b1174e1edd44637fadcdc282f3f92
+EBUILD prelude-manager-3.0.0.ebuild 1359 SHA256 a64d5abcf2b27d2601708717018bd163100df4dcf0e1327f61c82af17e56894b SHA512 40b02f22cf0c21975994b2f96e4955feb5cbc014e43178055cbcf8b416eb41aea0d7531a3f3464cdbf6e6c0dd672e9175c4a97d7b10770416b730c28d832aae2 WHIRLPOOL 03363e2fb35d4a674d2dd5d69510ad66ad430c724c12693d884f844a545567a85e189a1ed46849c367bf90868cff7cb5efc272ea6158702cdf4a79d9b0924191
```

Comment:
The manifest file shouldn't include checksums for the ebuild and metadata files. This typically happens if you had a local overlay where [`thin-manifests`](https://gitweb.gentoo.org/repo/gentoo.git/tree/metadata/layout.conf#n36) isn't enabled. `repoman manifest` will regenerate the manifest and remove unnecessary entries if you run it inside the git repo, or enable thin-manifests in your local overlay.